stage1/init/container.go

```diff
@@ -147,6 +147,10 @@ func (c *Container) appToSystemd(ra *schema.RuntimeApp, am *schema.ImageManifest
 		newUnitOption("Service", "Group", app.Group),
 	}
 
+	if interactive {
+		opts = append(opts, newUnitOption("Service", "StandardInput", "tty"))
```

Comment:
Can we add the same for StandardOutput? The default of StandardOutput is journal. I'm not sure how it works, maybe the journal is redirected to /dev/console, which systemd-nspawn has created as a pseudoterminal.